frontend/src/pages/login/resetPassword/ResetPassword.tsx

```diff
@@ -0,0 +1,149 @@
+      setErrorMessage('Passwords do not match');
+      return;
+    }
+
+    const basicAuthToken = getBasicAuth(token, password);
+
+    const request = await fetch('http://localhost:5000/user/reset/', {
+      method: 'POST',
+      headers: { Authorization: basicAuthToken }
+    });
+
+    const response = (await request.json()) as ErrorResponse | SuccessResponse;
+
+    if (!response.status) handleResetPasswordError(response);
+    else handleResetPasswordSuccess(response);
+  };
+
+  const getPasswords = (form: HTMLFormElement) => {
+    const formData = new FormData(form);
+    const password = formData.get('password') as string;
+    const confirm = formData.get('confirm') as string;
+    return { password, confirm };
+  };
+
+  const getBasicAuth = (password: string, confirm: string): string => {
+    const hash = btoa(`${password}:${confirm}`);
+    const token = `Basic ${hash}`;
+    return token;
+  };
+
+  const handleResetPasswordError = ({ error }: ErrorResponse) => {
+    setError(true);
+    setErrorMessage(error);
+  };
+
+  const handleResetPasswordSuccess = ({ message }: SuccessResponse) => {
+    setError(false);
+    setSuccessMessage(message);
+    setSuccess(true);
+  };
+
+  const handleClose = (event?: React.SyntheticEvent, reason?: string) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+    setSuccess(false);
+  };
+
+  return (
+    <div className="Login">
+      <Snackbar
+        anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+        open={success}
+        autoHideDuration={10000}
+        onClose={handleClose}>
+        <Alert onClose={handleClose} severity="success">
+          {successMessage}
+        </Alert>
+      </Snackbar>
+
+      <div className="login-card">
+        <p className="login-card__title lead">ERP Software</p>
+        <p className="login-card__header h4">Modify Password</p>
+
+        <form onSubmit={tryResetPassword} className="login-card__form">
+          <div className="login-card__form__email">
+            <TextField
+              name="password"
+              type="password"
+              variant="outlined"
+              label="New Password"
+              fullWidth
+              required
+              error={error}
+              InputLabelProps={{
+                shrink: true
+              }}
+            />
+          </div>
+          <div className="login-card__form__password">
+            <TextField
+              name="confirm"
+              type="password"
+              variant="outlined"
+              label="Confirm Password"
+              fullWidth
+              required
+              error={error}
+              helperText={error ? errorMessage : ''}
+              InputLabelProps={{
+                shrink: true
+              }}
+            />
+          </div>
+          <div>
+            <div className="login-card__form__bottom">
+              <Button variant="contained" color="primary" component={Link} to="/">
```

Comment:
``variant="contained"`` should be reserved for the main button of a page.

Two ``variant="contained"`` is bad UX because the user can interpret it as there being two "main action".

I acknowledge that I'm being nitpicky.
```suggestion
              <Button variant="outlined" color="primary" component={Link} to="/">
```

Reply:
Ok, I agree.